.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Jetzt hat es schon wieder eine Datei aus `.idea` hierrein geschafft - was ist denn da los? Ihr müsst euch nochmal eure `.gitignore` angucken. Außerdem, solltest du bewusster Dateien mit `git add` hinzufügen. Bei jeder einzelnen solltest du sicher sein, dass andere Team-Mitglider sie auch brauchen. Das trifft auf nichts in `.idea` und `target` zu.